internal/eval/eval.go

```diff
@@ -2745,3 +2748,14 @@ func evalJSONSTRLEN(args []string, store *dstore.Store) []byte {
 	}
 	return clientio.Encode(strLenResults, false)
 }
+
+func evalSELECT(args []string, store *dstore.Store) []byte {
+	var b []byte
+
+	if len(args) != 1 {
+		return diceerrors.NewErrArity("SELECT")
+	}
+
+	b = clientio.RespOK
+	return b
```

Comment:
You can directly return `clientio.RespOK` without first assigning it to a temporary variable.

Reply:
Sure, will do that. I was printing a log in between, then removed the log.